training/distributed_training/pytorch/data_parallel/bert/pytorch_smdataparallel_bert_demo.ipynb

```diff
@@ -22,7 +22,7 @@
     "6. Define training metrics\n",
```

Comment:
<p>Need to list all permissions to run this notebook. So far I found the followings to be necessary :</p><p><code>AmazonSageMakerFullAccess</code></p><p> <code>AmazonEC2ContainerRegistryFullAccess</code></p><p> <code>AmazonFSxFullAccess</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>